### DISCUSSION TITLE
The difference between build and development

### DISCUSSION THREAD

**Discussion by l1mindr** [vercel/next.js#40024] [Help]
Hello
How long has it been when I push my codes, when it is taken from the build project once again, the changes are not applied, or it is different from what I see in my system.
Of course, only on pages that run as ssr

What could be the reason? please guide me

**Comment by IgorKowalczyk** [comment]
Maybe check the site cache. Some things can be cached by the browser

**Comment by icyJoseph** [comment]
Do you mean after you deploy to your service provider? Could you explain a bit further? A fresh deploy should be able to override previous content cached by your browser, but perhaps there's more at play here? Are you using a PWA?
